src/modules/navigator/mission_block.cpp

```diff
@@ -312,7 +313,7 @@ MissionBlock::is_mission_item_reached()
 
 			}
 
-			if (dist >= 0.0f && dist <= mission_acceptance_radius
+			if (dist_xy >= 0.0f && dist_xy <= mission_acceptance_radius
```

Comment:
How come `dist` changed to `dist_xy` here? Is that on purpose?

Reply:
Yes. In the same condition, it also checks `dist_z < _navigator->get_altitude_acceptance_radius`. Before, the Z distance was accounted for twice, and the altitude acceptance radius parameter had no effect if it was larger than the mission acceptance radius (which was the case for at least rovers).

Reply:
Ok good, I think I agree with the change but it is hidden within a pull request for the vehicle status. So I suggest to either change the title of the pull request and make sure it's pointed out in the description for reviewers to see, or alternatively you could create a separate pull request with just that change.

I would probably just change the title of the pull request in this case.